shim/src/interpose.c

```diff
@@ -153,7 +157,7 @@ static void init_libc(void)
     }
 }
 
-static void init(void)
+static inline void init(void)
```

Comment:
To foce `inline` you may want to consider overloading the keyword with GCC's attribute.You may consider putting this in `cc.h`:

```
#define inline __attribute__((always_inline))
```